alerts/cluster-monitoring-operator/KubeDaemonSetMisScheduled.md

```diff
@@ -0,0 +1,68 @@
+# KubeDaemonSetMisScheduled
+
+## Meaning
+
+The `KubeDaemonSetMisScheduled` alert triggers when, over a defined time interval, an excessive number of daemonset pods are found running on nodes where they should not be. This typically occurs when a pod is scheduled on a node with a taint that doesn't have a corresponding toleration, or when a node is unhealthy or overloading, hindering pod scheduling.
+
+## Impact
+
+Excessive resource usage where they could be used by other applications
```

Comment:
Hmm, I'd say that the main impact is that the daemonset pod may not run on all the expected nodes so some functionalities like node metrics or DNS service may be degraded.